### PR TITLE
initial wbn screenshot preview

### LINKED ISSUE #567: Preview: wbn
Support WBN object previews. This is easier now that the street is previewable and is already made of NFTs.

### PR DESCRIPTION
**Pull request by chrislatorres**
Fix #567 -- this gives a preview better than the current orange placeholder, it takes the first glb inside of the wbn and uses that for the screenshot

### CODE REVIEW

File: screenshot.html [hunk 1]
```diff
@@ -250,11 +251,14 @@ <h3 id=screenshot-header></h3>
       const {url, hash, ext, type, dst} = parseQuery(decodeURIComponent(window.location.search));
 
       try {
-        const _loadGltf = async () => {
+        const _loadGltf = async (passedUrl) => {
+          if (!passedUrl) {
```
Comment (avaer) [inline screenshot.html:255]: Why would this happen? If some file is missing this doesn't seem like the place to make one up.

Comment (avaer) [reply screenshot.html:255]: I would move this defaulting outside of this function or drop it altogether.

**Review by avaer — CHANGES_REQUESTED** [review]
Just the one change, then I think this is good.

File: screenshot.html (hunk 1 above)
Comment (chrislatorres) [reply screenshot.html]: droped it

**Pull request merged**